src/Language/Haskell/TH/Lens.hs

```diff
@@ -1152,23 +1205,40 @@ _DataInstD
       reviewer (x, y, z, w, u, v) = DataInstD x y z w u v
       remitter (DataInstD x y z w u v) = Just (x, y, z, w, u, v)
       remitter _ = Nothing
+#else
+_DataInstD :: Prism' Dec (Cxt, Name, [Type], [Con], [Name])
+_DataInstD
+  = prism' reviewer remitter
+  where
+      reviewer (x, y, z, w, u) = DataInstD x y z w u
+      remitter (DataInstD x y z w u) = Just (x, y, z, w, u)
+      remitter _ = Nothing
+#endif
 
-_NewtypeInstD :: DataPrism' [Type] Con
+-- |
+-- @
+-- _NewtypeInstD :: 'Prism'' 'Dec' ('Cxt', 'Maybe' ['TyVarBndr'], 'Type', 'Maybe' 'Kind', 'Con', ['DerivClause']) -- template-haskell-2.15+
+-- _NewtypeInstD :: 'Prism'' 'Dec' ('Cxt', 'Name', ['Type'],              'Maybe' 'Kind', 'Con', ['DerivClause']) -- template-haskell-2.12 through 2.14
```

Comment:
I think this formatting goes wrong, as quotes are zero-space when rendered.

Reply:
Ugh. What is the idiomatic way to align things in Haddock, then?

Reply:
Or, more to the point, do you think it's even worth trying to align things at all?

Reply:
After lots of lots of counting, I believe everything should be lined up properly now.